Microsoft.DotNet.Try.Jupyter.Tests/HistorySerializationTests.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.DotNet.Try.Jupyter.Protocol;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Microsoft.DotNet.Try.Jupyter.Tests
+{
+    public class HistorySerializationTests
+    {
+        [Fact]
+        public void Can_serialize_input_and_output_element()
+        {
+            var element = new InputOutputHistoryElement(0, 0, "input value", "output result");
+            var serialized = JsonConvert.SerializeObject(element);
+            serialized.Should().Be(@"[0,0,[""input value"",""output result""]]");
```

Comment:
Verifying round-trip serializability for these tests will be a bit lower maintenance. For example, reordering fields will impact the JSON string but not the correctness.

Reply:
this this message type reordering will break things. so it must be respected